xdn-nx-example/XDN.md

```diff
@@ -0,0 +1,203 @@
+# XDN
+
+## Development: run NX app
+
+```sh
+yarn start
+```
+
+## Build
+
+```sh
+yarn xdn:build
+```
+
+## Production: run XDN app
+
+```sh
+yarn xdn:run
+```
+
+## Deployment: XDN
+
+```sh
+yarn xdn:deploy
+```
+
+# XDN: creating NX app connector guide
```

Comment:
Since this is part of https://github.com/moovweb-docs/xdn-docs/pull/125, do we need to documentation in the example?

Reply:
Yes, I just decided to leave it here as well in the project as a part of its readme (to let devs know on how to work with it)